src/test/java/mate/academy/dao/impl/UserDaoImplTest.java

```diff
@@ -0,0 +1,34 @@
+package mate.academy.dao.impl;
+
+import mate.academy.dao.UserDao;
+import mate.academy.model.Role;
+import mate.academy.model.User;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class UserDaoImplTest extends AbstractTest {
+    private UserDao userDao;
+    private User bob;
+
+    @Override
+    protected Class<?>[] entities() {
+        return new Class[]{User.class, Role.class};
+    }
+
+    @BeforeEach
+    void setUp() {
+        userDao = new UserDaoImpl(getSessionFactory());
+        bob = new User();
+        bob.setEmail("bob@i.ua");
+        bob.setPassword("1234");
+        bob.setId(1L);
+    }
+
+    @Test
+    void save_Ok() {
```

Comment:
```suggestion
    void save_ok() {
```
the same for other methods